gluster-exporter/metric_brick.go

```diff
@@ -190,6 +433,25 @@ func brickUtilization() error {
 					if brick.Type != glusterutils.BrickTypeArbiter && usage.Used >= maxBrickUsed {
 						maxBrickUsed = usage.Used
 					}
+					// Get lvm usage details
+					stats, err := lvmUsage(brick.Path)
+					if err != nil {
+						log.WithError(err).WithFields(log.Fields{
+							"volume":     volume.Name,
+							"brick_path": brick.Path,
+						}).Error("Error getting lvm usage")
+						continue
+					}
+					// Add metrics
+					for _, stat := range stats {
+						var lvmLbls = getGlusterLVMLabels(brick, subvol.Name, stat)
+						// Convert to bytes
+						glusterBrickLVSize.With(lvmLbls).Set(stat.Size * 1024)
```

Comment:
this is now in `KB`? `stat.Size * 1024 * 1024)`

Reply:
My bad. Will change and update